app/scripts/lib/sentry.js

```diff
@@ -71,6 +76,23 @@ define(function (require, exports, module) {
   }
 
   /**
+   * Determines if the error message should be sent
+   * Source: github.com/getsentry/raven-js/blob/0184ca3bc7624be0fb0b093d9a96becc424bc9b5/src/raven.js
+   *
+   * @param {Object} data Error message data
+   * @returns {Boolean} true if message should be sent.
+   */
+  function shouldSendCallback(data) {
+    if (data && data.message) {
+      var shouldSend = data.message !== PREVIOUS_ERROR;
```

Comment:
Is there a group of errors which should be whitelisted and always logged? 

A scenario I can think of attempting to login multiple times with an invalid password. This will only log the attempt once, however, it might be important to know that several attempts were made?